CHANGELOG.next.asciidoc

```diff
@@ -514,6 +514,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Unskip s3_request integration test. {pull}23887[23887]
 - Add system.hostfs configuration option for system module. {pull}23831[23831]
 - Fix GCP not able to request Cloudfunctions metrics if a region filter was set {pull}24218[24218]
+- Allow cgroup self-monitoring to see alternate `hostfs` paths {pull}24334[24334]
```

Comment:
If this option is going to be avaliable in all beats now, we would need to add something about this to the  _Affecting all beats_ section too. And probably also to the docs.

Reply:
Ah, good point. However, this only really *does anything* in metricbeat, I just realized...

Reply:
But self-monitor code is used by all beats, isn't it?

It looks good to me in any case if only Metricbeat is fixed here, and we decide later if we should use this in other beats.